routing_model/src/flows_src.rs

```diff
@@ -212,7 +208,6 @@ impl StartRelocateSrc {
     //
     // Routine state
     //
-
```

Comment:
Not sure about removing that space.
That block apply to all the functions bellow, not just the one?

Reply:
I'll add it back and the other two instances below.